nzedb/Console.php

```diff
@@ -420,10 +429,10 @@ public function updateConsoleInfo($gameInfo)
 		try {
 			$amaz = $this->fetchAmazonProperties($gameInfo['title'], $gameInfo['node']);
 		} catch (\Exception $e) {
-			if ($e->getMessage() == 'Invalid xml response.') {
+			if ($e->getMessage() === 'Invalid xml response.') {
 				$amaz = false;
 			} else {
-				throw new \Exception($e->getMessage(), $e->getCode(), $e);
+				throw new \RuntimeException($e->getMessage(), $e->getCode(), $e);
```

Comment:
This is potentially bad. If we re-throw it should be the same as was caught, changing the meaning would be confusing for debugging potential errors.